src/js/express.js

```diff
@@ -135,7 +149,21 @@ fluid.defaults("gpii.express", {
     gradeNames: ["fluid.modelComponent", "gpii.express.expressConfigHolder"],
     members: {
         directChildrenOfInterest: [],
-        childrenByParent:         {}
+        childrenByParent:         {},
+        views: {
```

Comment:
`gpii-handlebars` is also done.  I also renamed the function to reflect its more general use.